pkg/cmd/cli/cmd/project.go

```diff
@@ -227,6 +227,11 @@ func (o ProjectOptions) RunProject() error {
 		}
 	} else {
 		fmt.Fprintf(out, "Now using project %q on server %q.\n", namespaceInUse, clientCfg.Host)
+
+		// This means that we've just created a new project stanza.  take this opportunity to suggest oc status.
+		if contextNameIsGenerated {
+			fmt.Fprintln(out, "See more about your project using 'oc status'.")
```

Comment:
You can cheat and say "using the 'status' command"

On Mon, Jul 13, 2015 at 2:37 PM, David Eads <notifications@github.com>
wrote:

> In pkg/cmd/cli/cmd/project.go
> <https://github.com/openshift/origin/pull/3630#discussion_r34495913>:
>
> > @@ -227,6 +227,11 @@ func (o ProjectOptions) RunProject() error {
> >  		}
> >  	} else {
> >  		fmt.Fprintf(out, "Now using project %q on server %q.\n", namespaceInUse, clientCfg.Host)
> > +
> > +		// This means that we've just created a new project stanza.  take this opportunity to suggest oc status.
> > +		if contextNameIsGenerated {
> > +			fmt.Fprintln(out, "See more about your project using 'oc status'.")
>
> @smarterclayton <https://github.com/smarterclayton> some discussion about
> trying to pass a proper name all the way through: #3630 (comment)
> <https://github.com/openshift/origin/pull/3630#discussion_r34242336> . Is
> it valuable enough to do? It feel a little weird to plumb it all the way
> through into an Options style object. Constant for the preferred name?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/3630/files#r34495913>.
>



-- 
Clayton Coleman | Lead Engineer, OpenShift